src/main/java/org/jenkinsci/test/acceptance/controller/LocalController.java

```diff
@@ -209,12 +211,24 @@ protected void onReady() throws IOException {
     @Override
     public void stopNow() throws IOException {
         LOGGER.info("Stopping Jenkins (" + getLogId() + ") in " + this);
-        process.getProcess().destroy();
+        if (!process.getProcess().supportsNormalTermination() && SystemUtils.IS_OS_WINDOWS) {
+            new WinProcess(process.getProcess()).sendCtrlC();
+        } else {
+            // https://bugs.openjdk.org/browse/JDK-5101298
```

Comment:
https://bugs.openjdk.org/browse/JDK-5101298
Workaround thanks to @jglick from https://stackoverflow.com/a/78816768/12916